tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts

```diff
@@ -169,17 +199,17 @@ export class FromPixelsProgram implements WebGPUProgram {
       buffer: {type: 'storage' as const}
     });
     // Input buffer binding layout.
-    bindGroupLayoutEntries.push({
-      binding: 1,
-      visibility: GPUShaderStage.COMPUTE,
-      storageTexture: {access: 'read-only', format: 'rgba8unorm'}
-    });
+    this.useWgsl ?
+        bindGroupLayoutEntries.push(
+            {binding: 1, visibility: GPUShaderStage.COMPUTE, texture: {}}) :
+        bindGroupLayoutEntries.push({
+          binding: 1,
+          visibility: GPUShaderStage.COMPUTE,
+          storageTexture: {access: 'read-only', format: 'rgba8unorm'}
```

Comment:
Does it mean that `imageLoad` corresponds to `storageTexture` not `texture` in webgpu?

Reply:
In chromium textureLoad should be usable with either storage or non-storage textures.

However note in the spec, reading from storage textures has been removed entirely:
https://github.com/gpuweb/gpuweb/pull/1794
... until we have feedback that they are needed.
Please provide feedback if you're unable to use a non-storage texture binding here.

Reply:
> In chromium textureLoad should be usable with either storage or non-storage textures.

Yes, why we have this PR is due to glsl version shader reports warning about using deprecate built-in function. And I find that readonly storage texture is not supported in wgsl spec. So I use non-storage texture here.  
the tests shows that change to use non-storage texture works fine here.